### PR TITLE
Fix typo in 10.2 - :all-friends

### PR DESCRIPTION
**Pull request by holyjak**

### CODE REVIEW

File: DevelopersGuide.adoc [hunk 1]
```diff
@@ -5986,7 +5986,7 @@ A query has to be rooted somewhere, so we've invented a root-level keyword, `:al
 In Fulcro component queries that would look like this:
 
 ```
-[{:all-people (comp/get-query Person)}]
+[{:all-friends (comp/get-query Person)}]
```
Comment (awkay) [inline DevelopersGuide.adoc]: um...this does not match the text two lines earlier...is there some other part of the guide this is inconsistent with?

Comment (holyjak) [reply DevelopersGuide.adoc]: I overlooked that, I should fix that as well. But the text _below_ only refers to friends, not people.

**Comment by holyjak** [comment]
@awkay I fixed the PR so I think it is ready for merge now?

**Comment by awkay** [comment]
Thanks

**Pull request merged**